pkg/deploy/generator/scripts/rpVMSS.sh

```diff
@@ -101,6 +101,14 @@ yum -y install clamav azsec-clamav azsec-monitor azure-cli azure-mdsd azure-secu
   if [[ ${attempt} -lt 60 ]]; then sleep 30; else exit 1; fi
 done
 
+# Ensure podman's log driver is journald so that we don't fill up the drive
+# with logs
+# https://github.com/containers/common/blob/main/pkg/config/containers.conf
+cat >/etc/containers/containers.conf <<'EOF'
+[containers]
+log_driver = "journald"
+EOF
```

Comment:
question: elsewhere in this file, container systemd units are configured with:

```
-v /run/systemd/journal:/run/systemd/journal
```

does this change make that irrelevant, or would we need additional changes to our logger configuration at a golang level to stop "double logging" to journal (or, at the very least, implement principal of least privilege for host mounting of system files)?

Reply:
oh lol, you already left a comment about this elsewhere haha - I think this is worth fixing, if for nothing else, kusto query performance.